views/partials/menu.ejs

```diff
@@ -43,8 +43,8 @@
                         <li><a href="/projects">Search</a></li>
                         <% if (user.canEdit) { %>
                         <li role="separator" class="divider"></li>
-                        <li><a href="/projects/tags/edit">Tags</a></li>
                         <li><a href="/project/add">Add Project</a></li>
+                        <li><a href="/projects/tags/edit">Manage Tags</a></li>
```

Comment:
Issue #82 states the wording should be: 'Manage Project Tags'. Was this agreed to change to this?